wger/nutrition/templates/ingredient/overview.html

```diff
@@ -20,6 +20,12 @@
                 minChars: 3,
                 onSelect: function (suggestion) {
                      window.location.href = '/nutrition/ingredient/' + suggestion.data.id + '/view/'
+                },
+                formatResult: function (suggestion) {
+                    if (suggestion.data.image_thumbnail) {
+                        return '<div><img src="' + suggestion.data.image_thumbnail + '" /> ' + suggestion.value + '</div>';
+                    }
+                    return '<div>' + suggestion.value + '</div>';
```

Comment:
Can you add the same format result to wgerInitIngredientAutocompleter() as well? Then it'll be shown on the nutritional plan form as well

Reply:
Oh sure :+1: